components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -113,7 +113,7 @@ class SHOCMacrophysics : public scream::AtmosphereProcess
 
         // Temperature
         const auto& theta_zt = PF::calculate_theta_from_T(T_mid(i,k),p_mid(i,k));
-        thlm(i,k) = theta_zt - (latvap/cpair)*qc(i,k);
+        thlm(i,k) = theta_zt-(theta_zt/T_mid(i,k))*(latvap/cpair)*qc(i,k);
```

Comment:
Where would this code be in e3sm? Maybe that is a change i am missing. 

Also, if you are aware of other changes in e3sm that use thlm definition, please, let me know. So far i used it in 2 places in clubb_intr.

Reply:
You should only need to modify clubb_intr.F90 for consistent CLUBB modification (to my knowledge anyway).  If you can point me to your branch I can double check that your modifications are consistent.